.travis.yml

```diff
@@ -58,7 +61,7 @@ matrix:
         - SERVER_VERSION=3.6.5
 
 before_install:
-  - openssl aes-256-cbc -K $encrypted_b354efda2943_key -iv $encrypted_b354efda2943_iv -in .travis.scripts/atlas-uris.txt.enc -out .travis.scripts/atlas-uris.txt -d
+  - openssl aes-256-cbc -K $encrypted_b354efda2943_key -iv $encrypted_b354efda2943_iv -in .travis.scripts/atlas-uris.txt.enc -out .travis.scripts/atlas-uris.txt -d || true
```

Comment:
Does this belong in the commit for adding a sharded replica set? Looks like a fix for the automated Atlas testing ticket.